.browserslistrc

```diff
@@ -1,17 +0,0 @@
-# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
Can we investigate why this is removed in Angular 15? Migration stuff?

Reply:
Would be great to add explanation as comment here or in the PR description

Reply:
It was removed in Angular 15: 
 https://github.com/angular/angular-cli/releases/tag/15.0.0
 https://github.com/angular/angular-cli/commit/9beb878e2eecd32e499c8af557f22f46548248fc
 
 If we have custom need it should be re-added

Reply:
Cool, lets just leave this convo unresolved for future use